### ISSUE TITLE
Password flag should not be display

### ISSUE THREAD

**Issue by florian0410** [DataDog/ansible-datadog#404]
Hello guys,

I recently started to use this role to install my adatadog agent on Windows servers 2019

We cannot use the ":" character in the password as indicated in the [documentation](https://docs.datadoghq.com/fr/agent/faq/windows-agent-ddagent-user/#installation) so we use ansible to generate a password and specify it using `DDAGENTUSER_PASSWORD`. 

```yml
- hosts: default
  roles:
    - role: datadog.datadog
      datadog_windows_ddagentuser_name: "LocalSystem"
      datadog_windows_ddagentuser_password: "{{ lookup('password', '/dev/null' , chars='ascii_lowercase, ascii_uppercase, digits, hexdigits, octdigits, printable, whitespace') }}"
```

It results in the following execution at some point :

```bash
   amazon-ebs.this: TASK [datadog.datadog : Show installation flags] *******************************
    amazon-ebs.this: ok: [default] => {
    amazon-ebs.this:     "win_install_args": "  DDAGENTUSER_NAME=LocalSystem DDAGENTUSER_PASSWORD=PASSWORD_IN_CLEAR"
    amazon-ebs.this: }
    amazon-ebs.this:
```

This part of the code seems to be in cause.

https://github.com/DataDog/ansible-datadog/blob/14faa069e825335fdcf7eb0d104379015043459d/tasks/pkg-windows.yml#L66-L68 

Is there a way to specify safely the password to avoid print during ansible execution ?

**Comment by bkabrda** [comment]
Hi 👋 thanks for the report!
I agree that we need to fix this. However, I think it will require a more thorough fix so that the password doesn't get printed in logs at any point when we try to use this variable. I'll work on a fix.

**Comment by bkabrda** [comment]
Hi again 👋 I've implemented this in #415 and it will be part of the next release.
